app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_name = command.split()
+
+    if file_name[1] != file_name[2]:
+        if file_name[0] == "cp":
```

Comment:
no need to take values by index, u can use split method to unpack splitted string to variables
arg1, arg2, arg3 = ur_string.split()